src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.ts

```diff
@@ -278,7 +285,9 @@ export class CreateEditNewsComponent extends FormBaseComponent implements OnInit
 
   public toggleIsActive(filterObj: FilterModel, newValue: boolean): void {
     const index = this.filters.findIndex((item: FilterModel) => item.name === filterObj.name);
-    this.filters = this.filterArr({ name: filterObj.name, isActive: newValue }, index);
+    const changedtags = this.filterArr({ name: filterObj.name, isActive: newValue }, index);
+    this.filters = changedtags;
+    localStorage.setItem('newsTags', JSON.stringify(changedtags));
```

Comment:
we should use localStorageSerrvice for this

Reply:
done